lib/__tests__/standalone-formatter.test.mjs

```diff
@@ -57,3 +58,52 @@ it('standalone with input css and custom promised formatter', async () => {
 
 	expect(report).toContain('block-no-empty');
 });
+
+it('standalone with formatter specified in configuration', async () => {
+	const configWithFormatter = {
+		...configBlockNoEmpty,
+		formatter: 'string',
+	};
+
+	const { report } = await standalone({
+		code: 'a {}',
+		config: configWithFormatter,
+	});
+
+	const strippedOutput = stripAnsi(report);
+
+	expect(strippedOutput).toContain('1:3');
+	expect(strippedOutput).toContain('block-no-empty');
+});
+
+it('standalone with CLI formatter overriding configuration formatter', async () => {
```

Comment:
[suggestion] This code isn't actually related to CLI, avoiding the word seems better:

```suggestion
it('standalone with formatter argument overriding configuration formatter', async () => {
```